app/serializers/move_serializer.rb

```diff
@@ -51,12 +51,7 @@ class MoveSerializer < ActiveModel::Serializer
   }.freeze
 
   def person
-    # TODO: Remove the support for person id in future
-    if object.profile_id
-      object&.profile&.person
-    elsif object.person_id
-      Person.find(object.person_id)
-    end
+    object&.profile&.person
```

Comment:
Don't think you need the guard against `object` being nil - just the one on the `profile`.

Reply:
Feel free to use github code suggestions. Makes it easier